decidim-comments/app/commands/decidim/comments/create_comment.rb

```diff
@@ -46,6 +46,7 @@ def create_comment
 
         mentioned_users = parsed.metadata[:user].users
         send_mention_notifications(mentioned_users) if mentioned_users.any?
+        CommentCreation.publish(@comment, parsed.metadata)
```

Comment:
I know I recommended using something like `SettingsChange`, but since this service can only be used for this specific comment I don't think it's necessary to extract it to another class.

Reply:
As argumented for `subscribe` I would keep it.
Although it is not the case that `publish` will be reused, I think it is clear that if one of them is extracted the other one also should.

Reply:
Seems fair!